data_models/migrations/0024_populate_show_digas_id.py

```diff
@@ -31,13 +39,24 @@ def populate_digas_id(apps, schema_editor):
                 continue
 
             # We found something!
-            show.digas_id = digas_id_by_on_demand_link[sod_url]
+            digas_id = digas_id_by_on_demand_link[sod_url]
+            if digas_id in used_digas_ids: 
+                # Alert the user and don't update this show.
+                log.exception('Duplicate digas_id encountered for {show}.' + \
+                    'digas_id not updated.'.format(
+                        show=show.name,
+                    )
+                )
+                break
```

Comment:
We could just use `continue` here, to check with the next episode. That could ish work as a voting system, but would spam the terminal with the error message above if the error is not with a single episode of this show. So some system would need to be in place of only logging the error once per show, if we decide to `continue` here.

Reply:
Agreed, but we would then need a flag to be set. 

In my mind it reduces readability a little, while it will also trigger 15 messages for each show if the migration is rerun at a later point. The use case for this migration is a bulk update a single time, and will not be used at a later time (unless the migration is specifically rerun on a populated database). On database recreation, an existing database dump will be used, and the migration process will be run before the database is populated, so there won't be any shows to update. The shows will also have their digas_id already set, from the initial database dump. I suggest we leave it as is.